moq-pub/src/client.rs

```diff
@@ -0,0 +1,125 @@
+use crate::media::{self, MapSource};
+use http;
+use moq_transport::{Object, VarInt};
+use rustls;
+use rustls_native_certs;
+use std::io::Write;
+use std::net;
+use std::sync::Arc;
+use tokio::io::AsyncWriteExt;
+use tokio::task::JoinSet;
+use webtransport_quinn;
+
+use anyhow::Context;
+
+pub struct ClientConfig {
+	pub addr: net::SocketAddr,
+	pub uri: http::uri::Uri,
+}
+
+pub struct Client {
+	session: moq_transport_quinn::Session,
+	source: Arc<MapSource>,
+
+	recv_tasks: JoinSet<anyhow::Result<()>>,
+}
+
+impl Client {
```

Comment:
I don't think this is used?

Reply:
Oops, yes, this whole file was re-committed accidentally from an earlier version of the code.

- [ ] Remove client.rs